### PR TITLE
Stop services for nodes in parked state

### PR DESCRIPTION
**Pull request by hmusum**

### CODE REVIEW

File: node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java [hunk 1]
```diff
@@ -402,12 +402,15 @@ void doConverge(NodeAgentContext context) {
...
             case reserved:
-            case parked:
             case failed:
             case inactive:
                 removeContainerIfNeededUpdateContainerState(context, container);
                 updateNodeRepoWithCurrentAttributes(context);
                 break;
+            case parked:
+                updateNodeRepoWithCurrentAttributes(context);
+                stopServices(context);
```
Comment (hakonhall) [inline node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java]: This will log "Stopping services" on every tick.

Comment (hmusum) [reply node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java]: Yeah, good point, will add a stopServicesIfNeeded() and use that instead

**Review by hakonhall — APPROVED** [review]

**Comment by hakonhall** [comment]
We should probably do the same with inactive and failed?

**Comment by freva** [comment]
I think the criteria should be if the node no longer has an `owner`. This would also allow operator to manually park/fail a node (which will not clear the `owner`) without host-admin doing anything to the container.

**Comment by hmusum** [comment]
Sounds good to me, I'll create a PR implementing this

**Pull request merged**